src/commands/init/init.ts

```diff
@@ -0,0 +1,21 @@
+import path from 'path'
+
+import {
+  setupNpmProject,
+  setupGitIgnore,
+  setupDoxygen
+} from '../../utils/utils'
+
+import { createFolder } from '../../utils/file'
+import { createConfigFile } from '../shared/createConfigFile'
+
+export async function init() {
+  process.logger?.info('Creating folders and files...')
```

Comment:
It might be good to have a different message here than what we have for `sasjs create`.

Reply:
What should be better message on cli, @allanbowe 

> `Setting up environment for SASjs`
> `Placing minimum required files for SASjs`
> `Making it ready for SASjs`

;)

Reply:
How about `Initialising SASjs...`

Reply:
fixed